samples/excel/30-range/test-for-used-range.yaml

```diff
@@ -0,0 +1,125 @@
+id: excel-test-for-used-range
+name: Test for used range
+description: Create a chart from a table only if there's data in the table.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#try-create-chart-from-empty-table").click(() => tryCatch(tryCreateChartFromEmptyTable));
+
+        async function tryCreateChartFromEmptyTable() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");    
```

Comment:
I agree with this **""const" is just confusing relative to what it means in other programming languages."** But at this point we need to be consistent across all the snippets, so I'll stick with const for the most part.